pkg/kubelet/cm/cpumanager/policy_options.go

```diff
@@ -19,13 +19,42 @@ package cpumanager
 import (
 	"fmt"
 	"strconv"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	kubefeatures "k8s.io/kubernetes/pkg/features"
 )
 
 const (
 	// FullPCPUsOnlyOption is the name of the CPU Manager policy option
 	FullPCPUsOnlyOption string = "full-pcpus-only"
 )
 
+var (
+	alphaOptions = sets.NewString()
+	betaOptions  = sets.NewString(
+		FullPCPUsOnlyOption,
+	)
+	stableOptions = sets.NewString()
+)
+
+func CheckPolicyOptionAvailable(option string) error {
+	// from most likely to be true to less likely (here and in the rest of the function)
```

Comment:
This isn't actually in this order below. Here it is stable, beta, alpha. Below it is beta, alpha, stable. I think it might just make sense to order these the same as the sets above as alpha, beta, stable.